README.md

```diff
@@ -11,12 +11,20 @@ With the default flags you can run `eval $(glen -r)` to export your project's va
 
 ## Installation
 
-The easiest way to install glen is with [homebrew][]
+The easiest way to install glen is with [homebrew](https://brew.sh/)
```

Comment:
Markdown lets you define a matching link at the bottom of the file. The last line in this readme (which github won't let me comment on) already has this line and the link should work! This is fine too though and I'm happy to merge this.

Reply:
Oops, I didn't notice that. I've undone this change and used the same approach for the added asdf link to be consistent. Also noticed and fixed a couple of typos.